src/app/mod.rs

```diff
@@ -14,6 +14,7 @@ mod metric_labels;
 mod outbound;
 mod profiles;
 mod proxy;
+mod spans;
```

Comment:
Nit: I'm not convinced that `app` is the right place for this code...not a big deal though.